File: miniprogram/type/todo.ts
export type ToList = {
  id: string,
  iconColor : string,
  title: string,
  count: number,
  todos: Array<Todo>
}
export type Todo = {
  id: string,
  name: string,
  remark: string,
  listName: string,
  listId: string
  date: Date,
  flag: boolean,
  complete: boolean
}

export type TodoTop = {
  today: number,
  plan: number,
  all: number,
  flag: number
}

File: mini_cloud/mini_cloud/Databases/Entities/Todo.cs
using Newtonsoft.Json;

namespace mini_cloud.Databases.Entities;

public class Todo
{
     public Guid Id { get; set; }
     public string Name { get; set; }
     public string Remark { get; set; }
     public DateTime DateTime { get; set; }
     public bool Flag { get; set; }
     public bool Complete { get; set; }
     public Guid ToListId { get; set; }
     
     [JsonIgnore] 
     public ToList ToList { get; set; }
     
}